frontend/src/components/common/modal/confirmModal/ConfirmModal.tsx

```diff
@@ -0,0 +1,40 @@
+import Modal, { ModalProps } from "../Modal";
+import Button from "@/components/common/button/Button";
+import * as S from "@/components/common/modal/confirmModal/ConfirmModal.style";
+
+const ConfirmCustomStyle = {
+  display: "flex",
+  flexDirection: "column",
+  height: "fit-content",
+  gap: "5rem",
+  padding: "3rem",
+  fontSize: "1.6rem",
+  fontWeight: 700,
+};
+
+interface ConfirmModalProps extends ModalProps {
+  confirmButtonText?: string;
+  cancelButtonText?: string;
+}
+
+const ConfirmModal = ({
+  confirmButtonText = "확인",
+  cancelButtonText = "취소",
+  ...rest
+}: ConfirmModalProps) => {
+  return (
+    <Modal style={ConfirmCustomStyle as React.CSSProperties} {...rest}>
```

Comment:
```suggestion
    <Modal style={{ height: "fit-content" }} {...rest}>
```

이렇게 해서 하위에 새로  스타일로 나머지 flex, column 등등 만들어서 스타일 적용하는건 어떤가요?

Reply:
넵!! 나머지 속성들은 모달 자체에 주는 스타일이 아니라 모달 컨텐츠와 관련된 속성들이네요ㅎㅎ 변경하였습니다!